### PR TITLE
Fix threshold when raycasting with THREE.Points

### PR DESCRIPTION
**Pull request by Axiverse**
Currently, threshold isn't taken into account when quick rejecting a Points object by checking if the ray intersects with the bounding sphere, thus rejecting points on the edge of the bounding sphere or always if the Points objects has only one vertex.

Additionally changed Ray.intersectsSphere to compare length by the square radius to remove a sqrt function.

### CODE REVIEW

File: src/math/Ray.js [hunk 1]
```diff
@@ -276,9 +276,11 @@ THREE.Ray.prototype = {
 
 	}(),
 
-	intersectsSphere: function ( sphere ) {
+	intersectsSphere: function ( sphere, threshold ) {
```
Comment (WestLangley) [inline src/math/Ray.js]: I would not change this method. The other methods of this class do not have a tolerance parameter.

If this method is not doing what you want, then don't use it -- inline the code, instead.

**Comment by Axiverse** [comment]
Good point, that felt a little off doing that, inlining makes a lot more sense here.

**Comment by BrianSipple** [comment]
This PR looks to have fallen off the grid. @WestLangley, are these changes acceptable/desirable now after @Axiverse's latest update?

**Comment by WestLangley** [comment]
Yes, it looks good to me.

**Comment by Mugen87** [comment]
The threshold was added via 5cd8b0300cf6f6befdf2657994b622a2c9ed31ff over a year ago.

**Pull request closed without merging**